clippy_lints/src/lib.rs

```diff
@@ -447,59 +447,80 @@ pub fn register_plugins(store: &mut rustc_lint::LintStore, sess: &Session, conf:
     // begin deprecated lints, do not remove this comment, it’s used in `update_lints`
     store.register_removed(
         "clippy::should_assert_eq",
-        "`assert!()` will be more flexible with RFC 2011",
+        "This used to check for `assert!(a == b)` and recommend replacement with \
```

Comment:
These messages are displayed when the lint is used in the code. So we have to apply the same rules as to every other suggestion: start lowercase and don't put punctuation  at the end.

Those are also tested in our UI tests, so you will have to bless them.